package/kernel/linux/modules/netsupport.mk

```diff
@@ -1434,6 +1434,23 @@ endef
 $(eval $(call KernelPackage,inet-diag))
 
 
+define KernelPackage/inet-mptcp-diag
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=INET diag support for MultiPath TCP
+  DEPENDS:= +@KERNEL_MPTCP +@KERNEL_MPTCP_IPV6 +kmod-inet-diag
+  KCONFIG:= CONFIG_INET_MPTCP_DIAG@ge5.6
+  FILES:= $(LINUX_DIR)/net/mptcp/mptcp_diag.ko@ge5.6
```

Comment:
no `+` before the `@CONFIG_SYMBOLS`, `+` is for adding packages.

no need for the `@ge5.6` kernel 5.6 version switches after Kconfig symbol nor in file list (this branch now supports 5.15 and 6.1)